src/style.css

```diff
@@ -0,0 +1,81 @@
+* {
```

Comment:
- [ ] Also it would be better if our styling is improved to meet the design as shown in the screenshot below

![Screenshot from 2022-08-11 15-46-54](https://user-images.githubusercontent.com/46388113/184175467-2ba0714e-8fed-4b8f-8199-3e612686c5e0.png)

Reply:
I suggest this is not listed in the day 2 project.

Reply:
Well resolved 👍🏽